bin/node-template/node/src/service.rs

```diff
@@ -230,7 +230,7 @@ pub fn new_full(mut config: Configuration) -> Result<TaskManager, ServiceError>
 		let slot_duration = sc_consensus_aura::slot_duration(&*client)?;
 		let raw_slot_duration = slot_duration.slot_duration();
 
-		let aura = sc_consensus_aura::start_aura::<AuraPair, _, _, _, _, _, _, _, _, _, _>(
+		let aura = sc_consensus_aura::start_aura::<AuraPair, _, _, _, _, _, _, _, _, _, _, _>(
```

Comment:
:see_no_evil:

Reply:
All because keypair type can't be inferred. Life's hard.